### PR TITLE
Feat/ripple

### PR DESCRIPTION
**Pull request by williamernest**
fixes: #72

### CODE REVIEW

File: src/styles/RipplePage.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.ripple-demo-icon.mdc-ripple-surface {
+  @include mdc-ripple-radius-unbounded;
+
+  width: 24px;
+  height: 24px;
+  font-size: 24px;
+  padding: 12px;
+}
```
Comment (lynnjepsen) [inline src/styles/RipplePage.scss]: new line

File: src/RipplePage.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+import './styles/RipplePage.scss';
+
+const RipplePage = () => {
+  return (
+    <div>
+      <HeaderBar />
+      <ComponentPage
+        hero={<RippleHero />}
+        title='Ripple'
+        description=''
```
Comment (lynnjepsen) [inline src/RipplePage.js]: description:

Ripples are visual representations used to communicate the status of a component or interactive element.

File: src/RipplePage.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        designLink='https://material.io/guidelines/motion/choreography.html#choreography-radial-reaction'
+        docsLink='https://material.io/components/web/catalog/ripples/'
+        sourceLink='https://github.com/material-components/material-components-web/tree/master/packages/mdc-ripple'
+        demos={<RippleDemos />}
+      />
+    </div>
+  );
+}
+
+class RippleHero extends Component {
```
Comment (lynnjepsen) [inline src/RipplePage.js]: the hero ripple needs a bounding box, its not obvious where the ripple is right now.

File: src/RipplePage.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        <div>
+          <div className='ripple-demos'>
+            <div className='ripple-demo-col'>
+              <h3>{title}</h3>
+              <div className={variant + ' mdc-ripple-surface'} ref={this.initRipple}>
+                {text}
+              </div>
+            </div>
+            <div>
+              <h3>{title} - CSS Only</h3>
```
Comment (lynnjepsen) [inline src/RipplePage.js]: Drop the CSS only

File: src/RipplePage.js (hunk 3 above)
Comment (williamernest) [reply src/RipplePage.js]: It has a bounding box. It's the entire middle portion of the hero section.

**Pull request merged**